athena/WinConditions.tsx

```diff
@@ -162,7 +184,9 @@ export type WinConditionsWithVectors =
 
 export type WinCondition =
   | Readonly<{
+      completed?: PlayerIDSet;
       hidden: boolean;
+      optional: boolean;
```

Comment:
The default win condition should not have either of these fields, and should never be able to be optional. Please remove them (and then remove the places where you used those for the defaults like in `MapData`).

You might have to check for `condition.type !== WinCriteria.Default` in a few places to make TypeScript happy.

Reply:
Completed in 00a5794cd36ebfb7d957f3c5551d19f67db14b15